doc/classes/Tree.xml

```diff
@@ -288,6 +295,14 @@
 				Sets the title of a column.
 			</description>
 		</method>
+		<method name="set_column_title_alignment">
+			<return type="void" />
+			<param index="0" name="column" type="int" />
+			<param index="1" name="title_alignment" type="int" enum="HorizontalAlignment" />
+			<description>
+				Sets column title alignment.
```

Comment:
Seems like column titles don't support newlines, so `HORIZONTAL_ALIGNMENT_FILL` won't have effect here. Might be worth mentioning that maybe.

Reply:
Should setting `HORIZONTAL_ALIGNMENT_FILL` produce an error message?

Reply:
Maybe it could print a warning 🤔

Reply:
Perhaps something like this?
```C++
	if (p_alignment == HORIZONTAL_ALIGNMENT_FILL) {
		WARN_PRINT("HORIZONTAL_ALIGNMENT_FILL is not supported for column titles");
		return;
	}
```

Reply:
I think the warning should be in `set_text_alignment()` if anything.